packages/docusaurus-theme-classic/src/__tests__/__snapshots__/inlineScripts.test.ts.snap

```diff
@@ -0,0 +1,39 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`inlineScripts inline javascript for default options 1`] = `
+"(function() {
+    function getSystemColorMode() {
+      return window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
+    }
+
+    
+      function getQueryStringTheme() {
```

Comment:
Can we reduce line breaks and useless indentation?

This doesn't matter much (if minimization is enabled), but it looks a bit messy

Reply:
Done! Let me know if you're fine with that!

I did it this way before as my aim was to keep the source clean rather than the output. Now that we've prioritized a clean output, I feel the source looks a bit messy now, though.

I'm starting to see the appeal to [tc39/proposal-string-dedent](https://github.com/tc39/proposal-string-dedent).